### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by artyomwhite**
[DEMO LINK](https://artyomwhite.github.io/react_carousel/)

### CODE REVIEW

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Good job
Check this cases
https://github.com/mate-academy/react_carousel/assets/61904995/e696f026-5216-4bd1-8cab-4275976b3e35

File: src/App.tsx [hunk 1]
```diff
@@ -1,39 +1,52 @@
-import React from 'react';
+import { useState } from 'react';
+import { Carousel } from './Components/Carousel/Carousel';
+import { Form } from './Components/Form/Form';
```
Comment (IvanFesenko) [inline src/App.tsx:3]: Commit and push all files

Comment (IvanFesenko) [inline src/App.tsx:3]: Commit and push all files

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Issue still repro
https://github.com/mate-academy/react_carousel/assets/61904995/544e4673-9021-4605-a4fa-bb0138128d54

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
still not fixed issue from prev commetns
also commit and push all files
https://github.com/mate-academy/react_carousel/pull/456/files#r1273338251
https://github.com/mate-academy/react_carousel/pull/456/files#r1273205767

Screenshot from 2023-07-25 15-00-41
chrome-capture-2023-6-25

**Pull request closed without merging**